contrib/nydus-snapshotter/pkg/process/manager.go

```diff
@@ -250,7 +250,7 @@ func (m *Manager) Reconnect(ctx context.Context) error {
 
 	// cleanup database so that we'll have a clean database for this snapshotter process lifetime
 	log.L.Infof("found %d daemons running", len(daemons))
-	if err := m.store.CleanupDatabase(ctx); err != nil {
+	if err := m.store.CleanupDaemons(ctx); err != nil {
 		return errors.Wrapf(err, "failed to cleanup database")
```

Comment:
may be sync with the method name

Reply:
you mean it should add lock in CleanupDaemons？Maybe I misunderstand and  create new a branch use cleanup-daemons.

Reply:
> errors.Wrapf(err, "failed to cleanup database") 
sorry, I mean the error message should sync with the method name, it's trivial

Reply:
oh, I see, should fix next pr.